vanoce.php

```diff
@@ -87,6 +85,47 @@
         <strong>Celkem dárků: <?php echo count($darky);?></strong>
     </p>
 
+    <!-- úkol 2. -->
+    <h2>Příliš drahé dárky</h2>
+    <ul class="list-group my-3">
+        <?php foreach ($darky as $darek) {
+            if ($darek["cena"] > 1000) { ?>
+                <li class="list-group-item"><?php echo $darek["popis"]; ?></li>
+            <?php }
+        } ?>
+    </ul>
+
+    <!-- úkol 3. -->
+    <h2>Etikety</h2>
```

Comment:
Daval jsem tam jeste tohle, at je to hezci :upside_down_face: 

```suggestion
    <h2 class="mt-2">Etikety</h2>
```